integration/integration_test.go

```diff
@@ -2122,15 +2123,52 @@ func trustedClusters(t *testing.T, suite *integrationTestSuite, test trustedClus
 	sshPort, proxyWebPort, proxySSHPort := nodePorts[0], nodePorts[1], nodePorts[2]
 	require.NoError(t, aux.StartNodeAndProxy("aux-node", sshPort, proxyWebPort, proxySSHPort))
 
-	// wait for both sites to see each other via their reverse tunnels (for up to 10 seconds)
-	abortTime := time.Now().Add(time.Second * 10)
-	for len(checkGetClusters(t, main.Tunnel)) < 2 && len(checkGetClusters(t, aux.Tunnel)) < 2 {
-		time.Sleep(time.Millisecond * 2000)
-		require.False(t, time.Now().After(abortTime), "two clusters do not see each other: tunnels are not working")
+	// wait for main to see aux through a reverse tunnel
+	require.Eventually(t, func() bool {
+		site, err := main.Tunnel.GetSite(clusterAux)
+		if err != nil {
+			require.True(t, trace.IsNotFound(err))
+			return false
+		}
+		require.Equal(t, teleport.RemoteClusterStatusOnline, site.GetStatus())
+		return true
+	}, time.Second*10, time.Second*2, "main cluster doesn't have a reverse tunnel for aux")
+
+	// check that the CAs have been correctly synchronized between the two clusters
+	for _, c := range []struct {
+		instance *TeleInstance
+		caType   types.CertAuthType
+		name     string
+		trustRel types.TrustRelationship // empty if we expect the CA to not exist
+	}{
+		{main, types.HostCA, clusterMain, types.TrustRelationshipLocal},
```

Comment:
nit: use named initialization
```suggestion
		{instance: main, caType: types.HostCA, name: clusterMain, trustRel: types.TrustRelationshipLocal},
```